src/nvim/tui/tui.c

```diff
@@ -1470,9 +1468,8 @@ static void patch_terminfo_bugs(TUIData *data, const char *term,
         // per analysis of VT100Terminal.m
         || iterm || iterm_pretending_xterm
         || teraterm    // per TeraTerm "Supported Control Functions" doco
-        // Allows forcing the use of DECSCUSR on linux type terminals, such as
-        // console-terminal-emulator from the nosh toolset, which does indeed
-        // implement the xterm extension:
+        // Some linux-type terminals (such as console-terminal-emulator
+        // from the nosh toolset) implement implement the xterm extension.
```

Comment:
Perhaps a bit late now, But there is a typo here: "implement implement"

Reply:
thanks, fixed in "doc" PR.